src/statement_generator.cpp

```diff
@@ -393,10 +401,12 @@ unique_ptr<QueryNode> StatementGenerator::GenerateQueryNode() {
 	if (is_distinct) {
 		result->modifiers.push_back(make_uniq<DistinctModifier>());
 	}
-	if (RandomPercentage(20)) {
-		result->modifiers.push_back(GenerateOrderBy());
-	}
-	if (RandomPercentage(20)) {
+	if (verification_enabled) {
+		result->modifiers.push_back(GenerateOrderByAll());
+	} else if (!verification_enabled) {
```

Comment:
verification_enabled is a bool, so you don't need to check `if  (!verification_enabled)`

Reply:
oh, of course. Sorry for that